modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -391,23 +400,53 @@ def international_phone_too_long?
         phone.length > 25 if phone
       end
 
-      def validate_form_526_gulf_service
+      def validate_form_526_toxic_exposure
+        return if form_attributes&.dig('toxicExposure').nil? # nullable on schema
+
         gulf_war_service = form_attributes&.dig('toxicExposure', 'gulfWarHazardService')
-        return if gulf_war_service&.dig('servedInGulfWarHazardLocations') == 'NO'
+        validate_form_526_toxic_exp_sections(gulf_war_service, 'gulfWarHazardService')
+        herbicide_service = form_attributes&.dig('toxicExposure', 'herbicideHazardService')
+        validate_form_526_toxic_exp_sections(herbicide_service, 'herbicideHazardService')
+        other_exposures = form_attributes&.dig('toxicExposure', 'additionalHazardExposures')
+        validate_form_526_toxic_exp_sections(other_exposures, 'additionalHazardExposures')
+        multi_exposures = form_attributes&.dig('toxicExposure', 'multipleExposures')
+        validate_form_526_toxic_multi_exp(multi_exposures, 'multipleExposures')
+      end
+
+      def validate_form_526_toxic_exp_sections(section, attribute_name)
+        if section&.nil? || section&.dig('servedInHerbicideHazardLocations') == 'NO' ||
+           section&.dig('servedInGulfWarHazardLocations') == 'NO'
+          return
+        end
+
+        begin_date = section&.dig('serviceDates', 'beginDate')
+        end_date = section&.dig('serviceDates', 'endDate')
```

Comment:
Similar to multiple exposure dates, date section of `additionalHazardExposures` is defined as `exposureDates`, not `serviceDates`.